### PR TITLE
LPD-25885 - Warehouse Search doesn't function

### PR DESCRIPTION
**Pull request by victorcdp**
Issue: [LPD-25885](https://liferay.atlassian.net/browse/LPD-25885)
@andrea-ale-sbarra Can you review this? It's tied to an LPP

The problem is that there was no search param in the getWarehousesPage method

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:playwright

File: modules/test/playwright/tests/commerce/commerce-warehouse-web/warehouseSearch.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
+import {commercePagesTest} from '../../../fixtures/commercePagesTest';
+import {dataApiHelpersTest} from '../../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../../fixtures/loginTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/tests/commerce/commerce-warehouse-web/warehouseSearch.spec.ts:16]: this is not necessary anymore

File: modules/test/playwright/tests/commerce/commerce-warehouse-web/warehouseSearch.spec.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/tests/commerce/commerce-warehouse-web/warehouseSearch.spec.ts:10]: this is not necessary anymore

File: modules/test/playwright/tests/commerce/commerce-warehouse-web/warehouseSearch.spec.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		warehouse1.name['en_US']
+	);
+	await commerceAdminWarehousesPage.managementToolbarSearchInput.press(
+		'Enter'
+	);
+
+	await expect(page.getByText(warehouse1.name['en_US'])).toBeVisible();
+	await expect(page.getByText(warehouse2.name['en_US'])).toBeHidden();
+
+	apiHelpers.clearData();
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/tests/commerce/commerce-warehouse-web/warehouseSearch.spec.ts:43]: this is not necessary anymore

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
Code looks good, thank you!.
Can you please get rid of the unnecessary code?
Thank you!

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**